src/panels/config/script/ha-script-editor.ts

```diff
@@ -83,7 +102,75 @@ export class HaScriptEditor extends KeyboardShortcutMixin(LitElement) {
 
   @query("ha-yaml-editor", true) private _editor?: HaYamlEditor;
 
+  private _schema?: HaFormSelector[];
+
+  protected willUpdate(): void {
+    if (this._mode !== "gui" || !this._config) {
+      return;
+    }
+
+    const schema = BASE_GUI_SCHEMA;
+    if (!this.scriptEntityId) {
+      schema!.push({
```

Comment:
You keep pushing this to the base schema every update, you should not update the base schema